lib/scraper.rb

```diff
@@ -15,13 +15,13 @@ def provider=(title)
     end
 
     def provider_exists
-      unless self.content_provider
-        errors.add(:provider, 'not found')
-        errors.delete(:content_provider)
-      end
+      return if content_provider
```

Comment:
I see some editor autoformatting made its way into this pr. I'd be happy to un-add `lib/scraper.rb` if you'd like.